web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -60,6 +61,7 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     _addZAxisControls: function() {
 
         this.zAxisPickerContainer = new Portal.details.ZAxisPickerContainer({
+            id: 'zPicker',
```

Comment:
You cant specify 'id' here. There could be many zAxisPickerContainer containers

Reply:
append the id of this particular `NcWmsPanel.js` if its rendered at this stage ?